package.json

```diff
@@ -85,8 +85,8 @@
   "devDependencies": {
     "@chainsafe/blst": "^0.2.4",
     "@chainsafe/eslint-plugin-node": "^11.2.3",
-    "@chainsafe/lodestar-spec-test-util": "^0.18.0",
     "@chainsafe/threads": "^1.9.0",
+    "@lodestar/spec-test-util": "1.13.0",
```

Comment:
More recent versions require migration to `vitest`.

Reply:
Yes I am working on a way to generalize that, will be released soon.

Reply:
We just move this to vitest i would guess and use the same in all of our repos.  I am moving blst-ts to vitest in an upcoming PR